packages/compass-collection/src/components/workspace/workspace.tsx

```diff
@@ -1,26 +1,16 @@
-import type AppRegistry from 'hadron-app-registry';
-import React, { useCallback, useMemo } from 'react';
+import React, { useMemo } from 'react';
 import { connect } from 'react-redux';
+import { WorkspaceTabs, css, useHotkeys } from '@mongodb-js/compass-components';
 import {
-  WorkspaceTabs,
-  css,
-  cx,
-  useHotkeys,
-} from '@mongodb-js/compass-components';
-
-import {
-  createNewTab,
-  selectOrCreateTab,
-  closeTab,
-  prevTab,
-  nextTab,
-  moveTab,
-  selectTab,
-  changeActiveSubTab,
+  selectNextTab,
+  type CollectionTab,
```

Comment:
TIL. I did not know you could do this. I've been putting type imports on their own separate `import type {}` line 😆

Reply:
I think for a long time our linters/tooling were choking on this syntax anyway, glad to see that’s no longer the case :+1:

Reply:
I feel like this might be some (accidental?) eslint bump we did recently? Because this is eslint autofix that started using this syntax, I didn't do this manually 🤔